lib/rmt/scc.rb

```diff
@@ -21,8 +21,7 @@ def sync
 
     @logger.info(_('Updating products'))
     data = scc_api_client.list_products
-    data.each { |item| create_product(item) if (item[:product_type] == 'base') }
-    # with this loop, we create the migration paths after creating all products from sync, thus avoiding fk_constraint errors
+    data.each { |item| create_product(item) }
     data.each { |item| migration_paths(item) if (item[:product_type] == 'base') }
```

Comment:
Not sure if old logic was left here intentionally or not. Products without base won't have migrations, but maybe that's OK.

Reply:
does not apply to any of those product currently, but yeah it was not intentional
some products without base _could_ have migration path. fixed